app/Http/Controllers/EpisodeManager.php

```diff
@@ -23,9 +25,39 @@ class EpisodeManager extends Controller
     */
     public function index()
     {
-        $episodes = Episode::all();
 
-        return view('dashboard/pages/view_episodes', compact('episodes'));
+        $user                           = User::all();
+        $online_user                    = $user->where('active', 1)->count();
+        $offline_user                   = $user->where('active', 0)->count();
+        $numbers_of_users_on_suyabay    = $user->count();
```

Comment:
This should be `$total_number_of_users` instead of `$numbers_of_users_on_suyabay`